02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -86,7 +86,6 @@ visible: true
         * [max-length](#max-length-attribute)
         * [min-length](#min-length-attribute)
         * [parent-elements](#parent-elements-attribute)
```

Comment:
`parent-elements ` -- this one also should be removed

Reply:
removed `parent-elements`